docs/modules/05-assignment-5-aks-aca/02-aca/2-observability.md

```diff
@@ -0,0 +1,66 @@
+---
+title: (Optional) Observability
+parent: Deploying to Azure Container Apps
+grand_parent: Assignment 5 - Deploying to Azure with Dapr
+has_children: false
+nav_order: 2
+layout: default
+---
+
+# (Optional) Observability with Dapr using Application Insights
+
+In this section, you will deploy Dapr service-to-service telemetry using Application Insights. When [creating the Azure Container Apps environment](https://learn.microsoft.com/en-us/cli/azure/containerapp/env?view=azure-cli-latest#az-containerapp-env-create), you can set Application Insights instrumentation key that is used by Dapr to export service-to-service telemetry to Application Insights.
+
+## Step 1: Create Application Insights resource
+
+1. Create an Application Insights resource:
+
+    ```bash
+    az monitor app-insights component create --app cae-dapr-workshop-java --location eastus --kind web -g rg-dapr-workshop-java --application-type web
+    ```
+
+    You may receive a message to install the application-insights extension, if so please install the extension for this exercise.
+
+1. Get the instrumentation key for the Application Insights and set it to the `INSTRUMENTATION_KEY` variable:
```

Comment:
Should this bullet be 2? I see 1. in both. I will approve this PR for now, we can adjust it as a follow up.